_maps/runtimestation.json

```diff
@@ -3,6 +3,7 @@
     "map_path": "map_files/debug",
     "map_file": "runtimestation.dmm",
 	"shuttles": {
-	    "cargo": "cargo_delta"
+	    "cargo": "cargo_delta",
+		"exploration": "exploration_box"

```

Comment:
This is more indented than it is in the other files.

Reply:
Upon further review it looks like `exploration` should be under `shuttles` in all of the JSON files, which it isn't currently.